.github/workflows/healthchecks_clickhouse_ci.yml

```
@@ -0,0 +1,76 @@
+name: HealthChecks ClickHouse DB CI
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ master ]
+    paths:
+      - src/HealthChecks.ClickHouse/**
+      - test/HealthChecks.ClickHouse.Tests/**
+      - test/_SHARED/**
+      - .github/workflows/healthchecks_clickhouse_ci.yml
+      - Directory.Build.props
+      - Directory.Build.targets
+    tags-ignore:
+      - release-*
+      - preview-*
+
+  pull_request:
+    branches: [ master ]
+    paths:
+      - src/HealthChecks.ClickHouse/**
+      - test/HealthChecks.ClickHouse.Tests/**
+      - test/_SHARED/**
+      - .github/workflows/healthchecks_clickhouse_ci.yml
+      - Directory.Build.props
+      - Directory.Build.targets

```

We just need to adopt to https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks/pull/2330 (at the moment of writing you did the right thing, we just merged a PR that forces some changes)

```suggestion
      - Directory.Build.targets
      - Directory.Packages.props
    tags-ignore:
      - release-*
      - preview-*

  pull_request:
    branches: [ master ]
    paths:
      - src/HealthChecks.ClickHouse/**
      - test/HealthChecks.ClickHouse.Tests/**
      - test/_SHARED/**
      - .github/workflows/healthchecks_clickhouse_ci.yml
      - Directory.Build.props
      - Directory.Build.targets
      - Directory.Packages.props

```